.github/workflows/build-lint-test.yml

```
@@ -92,19 +103,14 @@ jobs:
         uses: actions/setup-node@v4
         with:
           node-version: 20
-      - name: Install dependencies
+      - name: Install npm dependencies
         run: npm ci
-      - name: e2e test install
+      - name: Install Playwright
         run: npx playwright install --with-deps
       - name: e2e test - lib ui
         run: npm run e2e:lib-ui
       - name: e2e test - app
```

```suggestion
      - name: Run app e2e tests
```
Maybe check the other names in this yml as well, so they are now consistently readable

---

Fixed in [b7dede0](https://github.com/kiesraad/abacus/pull/429/commits/b7dede0cfb3dd7e4400dc7988cd6ebb775881405)